test/unittests/parser_expr_test.cpp

```diff
@@ -355,27 +351,27 @@ TEST(parser_expr, instr_br_table)
         "04000000"  // label_count
         "00000000"  // arity
 
-        "23000000"  // code_offset
-        "88000000"  // imm_offset
+        "27000000"  // code_offset
+        "84000000"  // imm_offset
         "00000000"  // stack_drop
 
-        "1c000000"  // code_offset
-        "78000000"  // imm_offset
+        "20000000"  // code_offset
+        "74000000"  // imm_offset
         "00000000"  // stack_drop
 
-        "15000000"  // code_offset
-        "68000000"  // imm_offset
+        "19000000"  // code_offset
+        "64000000"  // imm_offset
         "00000000"  // stack_drop
 
-        "0e000000"  // code_offset
-        "58000000"  // imm_offset
+        "12000000"  // code_offset
+        "54000000"  // imm_offset
         "00000000"  // stack_drop
 
-        "2a000000"         // code_offset
-        "98000000"         // imm_offset
+        "2e000000"         // code_offset
+        "94000000"         // imm_offset
         "00000000"_bytes;  // stack_drop
 
-    EXPECT_EQ(code.immediates.substr(br_table_imm_offset, expected_br_imm.size()), expected_br_imm);
+    EXPECT_EQ(code.immediates.substr(0, expected_br_imm.size()), expected_br_imm);
```

Comment:
Why is this 0 now?

Reply:
Ah the old immediates table here skipped the local for some reason.

Reply:
This test always takes the part of the immediates for the `br_table` only. Now this part starts at 0 because we removed other immediates. I didn't bother to keep the `br_table_imm_offset` name. In the end when all immediates are move to instructions we check all bytes anyway.